examples/kotlin/shared/src/commonTest/kotlin/com/mongodb/realm/realmkmmapp/BundleTest.kt

```diff
@@ -0,0 +1,158 @@
+                    summary = "Do the laundry"
+                    isComplete = false
+                })
+
+                // Verify the data in the existing realm
+                // (this data should also be in the bundled realm we open later)
+                val originalItems: RealmResults<Item> = originalRealm.query<Item>().find()
+                for(item in originalItems) {
+                    Log.v("My Item: ${item.summary}")
+                }
+            }
+            // IMPORTANT: Sync all changes with server before copying the synced realm
+            originalRealm.syncSession.uploadAllLocalChanges(30.seconds)
+            originalRealm.syncSession.downloadAllServerChanges(30.seconds)
+
+            // Create a SyncConfiguration for the bundled copy with a file name
+            val copyConfig = SyncConfiguration.Builder(user, setOf(Item::class))
+                .name("bundled.realm")
+                .build()
+
+            // Delete any existing file before copying
+            Realm.deleteRealm(copyConfig)
+
+            // Copy the synced realm with writeCopyTo()
+            originalRealm.writeCopyTo(copyConfig)
+
+            // Get the path to the copy you just created
+            Log.v("Bundled realm location: ${copyConfig.path}")
+
+            // Verify the copied realm contains the data we expect
+            val copyRealm = Realm.open(copyConfig)
+            val copiedItems: RealmResults<Item> = copyRealm.query<Item>().find()
+            for(item in copiedItems) {
+                Log.v("My copied Item: ${item.summary}")
+            }
+
+            originalRealm.close()
+            copyRealm.close()
+            // :remove-start:
+            Realm.deleteRealm(originalConfig)
+            Realm.deleteRealm(copyConfig)
+            // :remove-end:
+        }
+        // :snippet-end:
+    }
+    @Test
+    fun bundleLocalRealm(){
+        // :snippet-start: bundle-local-realm
+        // Open an existing local realm
+        val originalConfig = RealmConfiguration.Builder(schema = setOf(Item::class))
+            .name("original.realm")
+            .build()
+        val originalRealm = Realm.open(originalConfig)
+
+        originalRealm.writeBlocking {
+            // Add seed data to the original realm
+            copyToRealm(Item().apply {
+                summary = "Do the dishes"
+                isComplete = false
+            })
+        }
+
+        // Create a RealmConfiguration for the bundled copy
+        val copyConfig = RealmConfiguration.Builder(schema = setOf(Item::class))
+            .name("bundled.realm")
+            .build()
+
+        // Delete existing file before copying
+        Realm.deleteRealm(copyConfig)
+
+        // Copy the realm data
+        originalRealm.writeCopyTo(copyConfig)
+
+        // Get the path to the copy you just created
+        Log.v("Bundled realm location: ${copyConfig.path}")
+        // :snippet-end:
+        originalRealm.close()
```

Comment:
```suggestion
        originalRealm.close()
        // :snippet-end:
```

might as well include closing the realm in here so it's a full runnable script